### PR TITLE
Feature/package management

### PR DESCRIPTION
**Pull request by sshelomentsev**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,56 +20,56 @@
...
-    "typescript": "^4.9.3",
-    "whatwg-fetch": "^3.6.2"
+    "dotenv": "^16.4.4",
+    "husky": "^8.0.3",
+    "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
+    "lint-staged": "^15.2.2",
+    "prettier": "^3.2.5",
+    "rimraf": "^5.0.5",
+    "rollup": "^4.11.0",
```
Comment (necipallef) [inline package.json:61]: The Rollup version is changed 2 MAJOR versions here, from v2 to v4. This is pretty aggressive. Why are we making this change, and how are we sure the whole build system has no problems?

This also applies to other packages too, but rollup is the first one I wanted to ask.

Comment (ilfa) [reply package.json:61]: The easiest way to find out is to compare artifacts generated with an old and with a new version.
I did it for the react SDK already and I was satisfied by the result.

**Review by ilfa — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (necipallef) [reply package.json:61]: Yes, but one also needs to read the changelog at least to make sure no unexpected changes are in the new version. My question is about sharing what has been done in terms of researching its effect.

Comment (necipallef) [reply package.json:61]: > The most important change for v4 is that minimal Node v18.

This is enough for me. Thanks!

**Review by necipallef — APPROVED** [review]

**Pull request merged**